bip32/src/prelude.rs

```diff
@@ -9,10 +9,13 @@ pub use crate::Bip32Error;
 pub use crate::defaults::*;
 
 /// Re-exported signer traits
-pub use k256::ecdsa::{
-    recoverable::Signature as RecoverableSignature,
-    signature::{DigestSigner, DigestVerifier, Signature as SigTrait},
-    Signature, SigningKey, VerifyingKey,
+pub use k256::{
```

Comment:
thoughts on
- re-export the whole crate `pub use k256`
- rename traits to dummy name `DigestSigner as _` so consumers don't need to know about them
- then re-export specifically useful things under regular name

Reply:
fixed.  I also made Signature as SigTrait a glob export.